src/nccl_ofi_net.c

```diff
@@ -950,8 +948,17 @@ static ncclResult_t ofi_connect(int dev, void *handle, void **sendComm)
 
 	/* Send "connect" message to remote EP */
 	do {
+		nccl_ofi_req_t *req = allocate_nccl_ofi_request(sComm->nccl_ofi_reqs_fl);
+
+		if (OFI_UNLIKELY(req == NULL)) {
+			ret = ncclSystemError;
+			NCCL_OFI_WARN("Unable to get NCCL OFI request for device %d",
+				     sComm->dev);
```

Comment:
Nit: Please align the argument to start of first line.

Reply:
Sorry, I'm not sure that got your suggestion. Should it be liek this
```
NCCL_OFI_WARN("Unable to get NCCL OFI request for device %d", sComm->dev);
```
?

Reply:
Thanks! I meant exactly what you did. I understand its not done everywhere and I intend to fix that :)